sm/engine/msm_basic/formula_imager_segm.py

```diff
@@ -85,8 +88,8 @@ def _gen_iso_images(spectra_it, sp_indexes, sf_peak_df, nrows, ncols, ppm, min_p
 
         # -1, + 1 are needed to extend sf_peak_mz range so that it covers 100% of spectra
         sf_peak_df = sf_peak_df[(sf_peak_df.mz >= sp_df.mz.min()-1) & (sf_peak_df.mz <= sp_df.mz.max()+1)]
-        lower = sf_peak_df.mz.map(lambda mz: mz - mz*ppm*1e-6)
-        upper = sf_peak_df.mz.map(lambda mz: mz + mz*ppm*1e-6)
+        lower = sf_peak_df.mz.map(lambda mz: mz - ABS_MZ_TOLERANCE_DA)
+        upper = sf_peak_df.mz.map(lambda mz: mz + ABS_MZ_TOLERANCE_DA)
```

Comment:
@iprotsyuk this change might cause the sci test to fail. Should be discussed with @andy-d-palmer first

Reply:
Yeah, I forgot to discuss it before sending the PR. Even if it doesn't break the tests, it's a conceptually important change. The best way to go would be to modify the config to allow for choosing between absolute and relative m/z tolerance

Reply:
Can you choose dynamically based on the datasets type (ims, lcms)?

Reply:
I can, but one may want to use ppm tolerance for LC-MS data. It's better to make it configurable

Reply:
Then it will require updates on both web app and metadata sides.